test/parallel_api/ranges/std_ranges_remove_if.pass.cpp

```diff
@@ -0,0 +1,34 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "std_ranges_test.h"
+
+std::int32_t
+main()
+{
+#if _ENABLE_STD_RANGES_TESTING
+    using namespace test_std_ranges;
+    namespace dpl_ranges = oneapi::dpl::ranges;
+
+    auto remove_if_checker = TEST_PREPARE_CALLABLE(std::ranges::remove_if);
+
+    test_range_algo<0>{big_sz}(dpl_ranges::remove_if, remove_if_checker, pred);
+    test_range_algo<1>{}(dpl_ranges::remove_if, remove_if_checker, pred, proj);
+//    test_range_algo<2, P2>{}(dpl_ranges::remove_if, remove_if_checker, pred, &P2::x);
+  //  test_range_algo<3, P2>{}(dpl_ranges::remove_if, remove_if_checker, pred, &P2::proj);
```

Comment:
Why is it commented out?

Reply:
I guess that is a kind of debugging  artefact...

Reply:
It's commented out again. Let me reopen this conversation.